packages/avatars/src/styled/StyledStatusIndicatorBase.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, keyframes } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { STATUS } from '../types';
+import {
+  TRANSITION_DURATION,
+  getStatusColor,
+  getStatusBorderOffset,
+  getStatusSize,
+  IStyledStatusIndicatorProps
+} from './utility';
+
+const COMPONENT_ID = 'avatars.status-indicator.base';
+
+const [offline] = [...STATUS].reverse();
```

Comment:
This feels brittle. I think you're better off just using the `'offline'` string below. TypeScript will keep things in check.